src/Traits/ReviewPermission.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\ContentReview\Traits;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Security;
+
+trait ReviewPermission
+{
+    /**
+     * Whether or not a user is allowed use content review
+     */
+    protected function canUseReviewContent(DataObject $record, ?Member $user = null): bool
+    {
+        // Whether or not the user is a reviewer. User must be allowed to view the page
+        $isReviewer = $record->canView($user) &&
+            $record->hasMethod('canBeReviewedBy') &&
+            $record->canBeReviewedBy($user);
```

Comment:
How come we have a `$record->canBeReviewedBy($user)` method and a `$record->canReviewContent($user)` method below?  Is that intentional? Seems like we'd only want  `$record->canReviewContent($user)` ?

Reply:
Agreed because canBeReviewedBy is already being called on the Review form we don't need to duplicate this check.